spec/capDL/Endpoint_D.thy

```diff
@@ -355,7 +355,7 @@ where
             else
               return ()
         | Some tcb_id_receiver \<Rightarrow> do
-             \<comment> \<open>liftM instead of bind+return avoids early unfolding in send_ipc_corres\<close>
+             \<comment> \<open>liftM instead of bind+return avoids early unfolding in sendIPC_corres\<close>
```

Comment:
This is probably a mistake, since the capDL spec goes upward from the abstract spec, not downward.

Reply:
Alright, will re-edit the `send_ipc_corres` commit to remove the changes in the `capDL` file.

Reply:
Does that mean we have a refinement from capDL to abstract with a lemma named `send_ipc_corres`?

Reply:
> Does that mean we have a refinement from capDL to abstract with a lemma named `send_ipc_corres`?

It appears to be called [`send_sync_ipc_corres`](https://github.com/seL4/l4v/blob/e019b90d8a80bfd65e0bfb51270b2d7edae7b761/proof/drefine/Ipc_DR.thy#L2632-L2640) :-)

I'd say that's out of scope for this PR, so perhaps just revert this change.

Reply:
Do you mean the change to the `capDL` file or the whole renaming for the `send_ipc_corres` commit?

Reply:
I mean that you should:
- revert this one-line change in `spec/capDL/Endpoint_D.thy`
- keep the rest of the renaming of `send_ipc_corres` to `sendIPC_corres`.

The problem is that `send_ipc_corres` in this comment is referring to a lemma in DRefine, *not* the lemma you are renaming in Refine. As it happens, the DRefine lemma to which this comment refers is actually called `send_sync_ipc_corres`, so the comment is wrong, but fixing that is out of scope for this PR.